app/build.gradle

```diff
@@ -51,7 +51,7 @@ android {
 dependencies {
     implementation 'com.android.support:multidex:2.0.1' // noinspection GradleDependency
     implementation 'androidx.core:core-ktx:1.9.0' // Default Android stuff
-    implementation 'androidx.appcompat:appcompat:1.5.1'
+    implementation 'androidx.appcompat:appcompat:1.6.0'
```

Comment:
Lets make a note of these version bumps in the release notes for the PR. Will help to track down this PR in the future in case a bug gets introduced and we need to rollback.

Reply:
Done.